recipes/mlrose/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mlrose" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cec83253bf6da67a7fb32b2c9ae13e9dbc6cfbcaae2aa3107993e69e9788f15e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
+    - scipy
+    - scikit-learn
+    - python >=3
+    - pip
+  run:
+    - python >=3
+
+test:
+  imports:
+    - mlrose
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/gkhayes/mlrose
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
The `LICENSE` is in the GitHub repo but not in the sdist archive.
Please open an issue in the upstream repo so it's part of next version.

For now, you should copy that [file](https://github.com/gkhayes/mlrose/blob/master/LICENSE) here (same directory as the meta.yaml file).